### ISSUE TITLE
Execution time printing

### ISSUE THREAD

**Issue by hbdeshmukh** [pivotalsoftware/quickstep#214]
Right now the execution time of each query is printed as follows:
``Execution time: 3.23487 seconds``

I have two proposals, fairly easy to implement. But before implementing them I want to take others' opinion: 
1. Do we need this high level of precision in printing the time? I think we can deal with 3 decimal places. A potential downside is for queries which run very fast (e.g. Create) will show 0.000 seconds. 
2. I have seen @pateljm doing post-processing of the output to remove Execution time and seconds, to copy in a spreadsheet. Do we need to get rid of the words and just print the number? If we remove them, the system will simply print a list of numbers, which we can quickly copy to a spreadsheet.

**Comment by hakanmemisoglu** [comment]
@hbdeshmukh Does #216 help?

**Comment by pateljm** [comment]
Good point Harshad! I have been thinking of switching to printing the time in ms and then keeping 3 decimal points. I believe this mirrors what PostgreSQL does. We can also change the format to simply be 
`Time: 15561.436 ms`. 

Ideally, we should also print out the number of tuples in the result. We are missing that info. A format like `(150 rows`), again similar to PostgreSQL would do.  

Hakan -- yup #216 is a good start. 

A somewhat related issue is that our output printing format sometime leaves a lot of extra whitespace, and the title "bar" is oddly formatted -- it is left-aligned while the actual values are right-aligned. We should right-align both, and when it is safe reduce the number of padded whitespaces.
